control-plane/agents/src/bin/core/controller/grpc.rs

```diff
@@ -131,6 +136,10 @@ impl MayaClientV1 {
     pub(crate) fn nexus(self) -> NexusClient {
         self.nexus
     }
+    /// Get the v1 pool client.
+    pub(crate) fn pool_client(&self) -> PoolClient {
```

Comment:
Can we change this to `pool` to maintain the similar nomeclature.

Reply:
Incorporated